ports/curl/portfile.cmake

```diff
@@ -109,7 +109,7 @@ if (VCPKG_WINDOWS)
 endif ()
 
 set(USE_OPENSSL ON)
-if (NOT ssl IN_LIST FEATURES)
+if (NOT libressl IN_LIST FEATURES)

```

Comment:
If the idea is to potentially support other backends, it's probably good to change this structure to make it clearer for someone else how to do so.